packages/alfa-rules/test/sia-r69/rule.spec.tsx

```diff
@@ -395,6 +395,40 @@ test(`evaluate() is inapplicable to the text that is part of a label of a disabl
   t.deepEqual(await evaluate(R69, { document }), [inapplicable(R69)]);
 });
 
+test("evaluate() is inapplicable to text that would otherwise pass if it was not only punctuation", async (t) => {
+  const target = h.text(",./;'[]\\-=?<>:\"{}|_+!@#$%^&*()");
+
+  const document = h.document([
+    <html
+      style={{
+        backgroundImage: "url('foo.png')",
+        color: "#000",
+      }}
+    >
+      {target}
+    </html>,
+  ]);
+
+  t.deepEqual(await evaluate(R69, { document }), [inapplicable(R69)]);
+});
+
+test("evaluate() is inapplicable to text that would otherwise fail if it was not only punctuation", async (t) => {
+  const target = h.text(",./;'[]\\-=?<>:\"{}|_+!@#$%^&*()");
+
+  const document = h.document([
+    <html
+      style={{
+        backgroundImage: "url('foo.png')",
```

Comment:
Same here. Which gets even stranger since the test case is now exactly the same.

Reply:
I based the tests on two other tests that were passing and failing resp., but I forgot to copy the oracle as well. Now I have based the tests on two other tests that are not using an oracle at all to keep it simple. I've confirmed that adding a single letter to the text in both tests results in passing and failing outcome resp.